Source/Find_Number_follow_Kaprekar_Constant.cpp

```diff
@@ -0,0 +1,156 @@
+#include <iostream>
+#include <vector>
+#include <climits>
+using std::swap;
+using std::vector;
+using std::cout;
+using namespace std;
+
+int partitionOfDigits(vector<long long int> &digitOfNumber, int lowestIndex, int highestIndex)
```

Comment:
In place of partitionOfDigits , you can use seperate_digits or number_to_array_conversion